imap_processing/lo/l1c/lo_l1c.py

```diff
@@ -484,3 +491,98 @@ def create_datasets(
             )
 
     return dataset
+
+
+def set_background_rates(
+    pointing_start_met: float,
+    pointing_end_met: float,
+    species: FilterType,
+    anc_dependencies: list,
+    attr_mgr: ImapCdfAttributes,
+) -> xr.DataArray:
```

Comment:
I'm surprised this passed mypy since it is returning a tuple of DataArrays and not a single one.

As a potential follow-up later, I'm wondering if we should switch these over to be a `updated_dataset = process_function(input_dataset, dependencies)`. So you would do whatever you needed to the input dataset rather than having to extract the pointing start/end beforehand.

Also, you are passing in attr_mgr here, but in other places in Lo I think you're doing it all at the end. So create the entire dataset, then at the end iterate through the variables you created and grab the metadata associated with those fields.

```suggestion
    pointing_start_met: float,
    pointing_end_met: float,
    species: FilterType,
    anc_dependencies: list,
    attr_mgr: ImapCdfAttributes,
) -> Tuple[xr.DataArray, xr.DataArray, xr.DataArray]:
```